package.json

```diff
@@ -65,20 +65,20 @@
     "uuid": "^8.3.1"
   },
   "devDependencies": {
-    "@adfinis-sygroup/semantic-release-config": "2.1.4",
+    "@adfinis-sygroup/semantic-release-config": "3.0.1",
     "@babel/plugin-proposal-object-rest-spread": "7.12.1",
     "@ember/jquery": "1.1.0",
     "@ember/optional-features": "2.0.0",
     "@glimmer/component": "1.0.2",
     "babel-eslint": "10.1.0",
     "broccoli-asset-rev": "3.0.0",
-    "ember-cli": "3.22.0",
+    "ember-cli": "3.21.2",
```

Comment:
Why downgrade?

Reply:
So its equal with ember-source and the ember cli update command works for the next update

Reply:
Okay, next time open a separate PR for this. This doesn't have anything to do with pinning ember-truth-helpers